packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1544,12 +1550,23 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
 
   @override
   void handleEvent(PointerEvent event, BoxHitTestEntry entry) {
-    if (ignorePointer)
-      return;
     assert(debugHandleEvent(event, entry));
-    if (event is PointerDownEvent && onSelectionChanged != null) {
-      _tap.addPointer(event);
-      _longPress.addPointer(event);
+    if (event is PointerDownEvent) {
+      // Checks if there is any gesture recognizer in the text span.
+      final Offset offset = entry.localPosition;
+      _layoutText(minWidth: constraints.minWidth, maxWidth: constraints.maxWidth);
```

Comment:
why do we call _layoutText during hit testing?

Reply:
i guess the theory is that this is cheap in the common case and it means we don't have to worry about people calling intrinsic size logic, etc, at unexpected times

Reply:
I think i should replace this with`assert(!debugNeedsLayout);`